pyproject.toml

```diff
@@ -72,7 +89,7 @@ repair-wheel-command = ""
 repair-wheel-command = ""
 
 [tool.pytest.ini_options]
-addopts = "--benchmark-skip"
+addopts = "-m 'not benchmark'"
```

Comment:
I'm not clear why this flag works.
Can you tell me a reference or example?

Reply:
Bechmarks are a custom pytest mark
https://github.com/spglib/spglib/blob/06f1799214e8caeb764b7d8435e0181d6fe215cf/python/test/test_benchmark.py#L11
So we can use built-in pytest mark filtering instead of `pytest-benchmark` cli

https://doc.pytest.org/en/latest/example/markers.html#marking-test-functions-and-selecting-them-for-a-run

I think we can also use a skip mark to always skip unless manually specified